backend/gradle/libs.versions.toml

```diff
@@ -62,7 +62,8 @@ org-junit-jupiter = { module = "org.junit.jupiter:junit-jupiter", version.ref =
 org-junit-jupiter-params = { module = "org.junit.jupiter:junit-jupiter-params", version.ref = "junit-jupiter" }
 org-piwik-java-tracking-matomo-java-tracker = "org.piwik.java.tracking:matomo-java-tracker:3.4.0"
 org-postgresql-postgresql = "org.postgresql:postgresql:42.7.6"
-org-simplejavamail = "org.simplejavamail:simple-java-mail:8.12.6"
+# 2273 occurs with 8.6.0 and higher
```

Comment:
🔧  The complete link to the issue would be more helpful

Reply:
this will be removed with an already open pr